_data/projects/faas.yml

```
@@ -0,0 +1,15 @@
+---
+name: faas
```

This name looks very general. Should it be more specific?

```suggestion
name: OpenFaaS
```